src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -76,6 +77,26 @@ public void Initialize(IModel channel, string mainQueue)
             channel.QueueBind(mainQueue, mainQueue, string.Empty);
         }
 
+        public string SetupDelay(IModel channel, long delay)
+        {
+            channel.ExchangeDeclare("delay-triggered", "fanout", useDurableExchanges);
+            channel.QueueDeclare("delay-triggered", useDurableExchanges, false, false, null);
+            channel.QueueBind("delay-triggered", "delay-triggered", "");
+
+            var address = $"delay-{delay}";
+
+            channel.ExchangeDeclare(address, "fanout", useDurableExchanges);
+
+            var arguments = new Dictionary<string, object>();
+            arguments.Add("x-message-ttl", delay);
+            arguments.Add("x-dead-letter-exchange", "delay-triggered");
+            channel.QueueDeclare(address, useDurableExchanges, false, false, arguments);
```

Comment:
Are we setting up a queue per each delay value passed? Is it possible to have a single queue and use per-message TTL?